fe/src/api/phones.ts

```diff
@@ -0,0 +1,7 @@
+import axios from 'axios';
+
+axios.defaults.baseURL = 'https://project-nice-gadgets.herokuapp.com/';
+
+export function getPhones() {
+  return axios.get('/products');
+}
```

Comment:
You also need to implement a function that adds a query to the request url

Reply:
> You also need to implement a function that adds a query to the request url

![IMAGE 2022-11-04 14:15:53](https://user-images.githubusercontent.com/87187483/199970476-995aad44-cc55-4fd0-b75a-5621fcf00835.jpg)